seeds/site_data.js

```diff
@@ -12,6 +15,35 @@ export async function seed(knex) {
 		if (Object.prototype.hasOwnProperty.call(seedSiteData, key)) {
 			let value = seedSiteData[key];
 			let data_type = typeof value;
+
+			// Remove attribution from metaTags for initial db seeding (if white-labeled)
+			if (key === "metaTags" && data_type === "object" && Array.isArray(value) && isWhiteLabeled) {
+				value.forEach(item => {
+					if (item.key === "twitter:site" || item.key === "twitter:creator") {
+						item.value = "kener";
+					}
+				});
+			}
+
+			// Remove Documentation & Github nav links from initial db seeding (if white-labeled)
+			if (key === "nav" && data_type === "object" && Array.isArray(value) && isWhiteLabeled) {
```

Comment:
we should keep github  and documentation as it will help people after they do 1 click deploy